src/plugins/visualize/public/application/components/visualize_listing.tsx

```diff
@@ -172,11 +174,13 @@ export const VisualizeListing = () => {
 
   return (
     <>
-      {dashboard.dashboardFeatureFlagConfig.allowByValueEmbeddables && (
-        <div className="visListingCallout">
-          <EuiCallOut size="s" title={calloutMessage} iconType="iInCircle" />
-        </div>
-      )}
+      {dashboardCapabilities.show &&
+        dashboardCapabilities.createNew &&
```

Comment:
Seems like this is overlapping with https://github.com/elastic/kibana/pull/89979

Reply:
Good call!